tests/opkg.py

```diff
@@ -36,7 +36,8 @@ def runTest(self):
 class OpkgUpdate(rootfs_boot.RootFSBootTest):
     '''Opkg is able to update list of packages.'''
     def runTest(self):
-        board.sendline('\nopkg update && echo "All package lists updated"')
+        board.sendline("\nopkg update | tee opkg.txt && ! grep 'Signature check failed' opkg.txt"
+                        "&& echo 'All package lists updated' && rm opkg.txt")
```

Comment:
Put it in recover, it won't really work even if I move it outside the sendline since I'd have to put it at the end of the function, which kind of defeats the purpose.